docs/usage/configuration-options.md

```diff
@@ -2754,3 +2754,13 @@ To disable the vulnerability alerts functionality completely, configure like thi
   }
 }
 ```
+
+To enable auto merge, configure like this:
```

Comment:
```suggestion
<!-- prettier-ignore -->
!!! tip
    We strongly recommend you have good test coverage before you automerge vulnerability alerts.

To automatically merge vulnerability alert PRs, configure like this:
```

I've rewritten the sentence, and put a `tip` admonition so that users know they should have good test coverage before enabling automerge.